org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/InitHandlerTest.java

```diff
@@ -121,6 +137,8 @@ private InitializeResult initialize(boolean dynamicRegistration) throws Interrup
 		workspaceCapabilities.setExecuteCommand(executeCommand);
 		capabilities.setWorkspace(workspaceCapabilities);
 		TextDocumentClientCapabilities textDocument = new TextDocumentClientCapabilities();
```

Comment:
need to set the Synchronization or else it NPEs

Reply:
This is not the only problem, :flushed:
I am trying to fix them